packages/status-js/package.json

```diff
@@ -38,7 +40,8 @@
     "protons-runtime": "^1.0.4"
   },
   "devDependencies": {
-    "protons": "^3.0.4"
+    "protons": "^3.0.4",
+    "ts-proto": "^1.115.1"
```

Comment:
⚠️ Should be rebased on https://github.com/status-im/status-web/pull/337 once merged instead.

Reply:
Also, `push_notification.proto` must be regenerated then.